packages/core/android/gradle.properties

```diff
@@ -1,4 +1,4 @@
-DdSdkReactNative_kotlinVersion=1.7.21
+DdSdkReactNative_kotlinVersion=1.8.21
```

Comment:
what is the error you are getting if Kotlin 1.7 is kept?

Reply:
After all the changes, reverting back to 1.7.21 does not seem to cause any issues now. I will look for the cause, and in the meantime I am running tests on the CI for all versions to make sure everything still works fine.

Reply:
For reference, 1.7.21 breaks our internal core package build and I could not identify the cause just yet. Since it is not a blocker, I will consider this done for now, by keeping v1.8.21, and keep a note for investigating this further in the future.